fitting/fitScanData.py

```diff
@@ -169,12 +239,11 @@ def fit(self, debug=False):
                 stepN = 0
                 
                 if debug:
-                    print "| stepN | vfatN | vfatCH | isVFAT3 | p0_low | p0 | p0_high | p1_low | p1 | p1_high | p2_low | p2 | p2_high |"
-                    print "| ----- | ----- | ------ | ------- | ------ | -- | ------- | ------ | -- | ------- | ------ | -- | ------- |"
-                while(stepN < 30):
-                    #rand = max(0.0, random.Gaus(10, 5)) # do not accept negative numbers
-                    rand = abs(random.Gaus(10, 5)) # take positive definite numbers
-
+                    print "| p0_low | p0 | p0_high | p1_low | p1 | p1_high | p2_low | p2 | p2_high |"
+                    print "| ------ | -- | ------- | ------ | -- | ------- | ------ | -- | ------- |"
+                while(stepN < 15):
+                    rand = max(0.0, random.Gaus(10, 5)) # do not accept negative numbers
+                    
```

Comment:
For sure this is not correct as for the case of `vfat3` it will cause only 1/2 of the `CAL_DAC` register to be scanned.  Double check your commit history and revert this; you may need to `cherry-pick` to remove bad commits or resolve this issue.